services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpoint.java

```diff
@@ -29,7 +30,7 @@
      */
     @GET
     @Path("list")
-    List<MigrationProjectAndAppCount> getMigrationProjects();
+    List<MigrationProjectAndAppCount> getMigrationProjects(@QueryParam("includeProvisional") Boolean includeProvisional);
```

Comment:
As I wrote above "This is not a normal PR, please do not comment what I didn't" - by which I mean, I pinpointed those which I suggest to discuss and merge. The rest is just rebase garbage.